### ISSUE TITLE
مشکل در تبدیل تاریخ فارسی

### ISSUE THREAD

**Issue by Sadeghbayan** [AminRahimi/angular-bootstrap-persian-datepicker#6]
سلام
بابت زحماتی که کشیدید خیلی خیلی ممنونیم .
یک مشکلی که هست اینه که وقتی که تاریخ رو به صورت دستی وارد می کنیم و بعد پاپ آپ رو باز می کنم تاریخ رو درست نشون نمیده .

من دقیقا مثل فرمتی که خودش هست به صورت دستی تاریخ رو وارد می کنم و در تقویم درست تبدیل نمیشه ، اما در نسخه انگلیسی این مشکل رو نداشتم .
ممنون میشم اگر راهنمایی کنید ؟

**Comment by AminRahimi** [comment]
سلام دوست عزیز
این مشکل به خاطر اینه که من از
parser
خود انگولار استفاده می کنم و اونم شمسی حالیش نیست
:)
persian-datepicker-tpls.jsخط زیر در فایل 
function parseDate(viewValue) {

**Comment by Sadeghbayan** [comment]
خب شما راه حلی ندارید ؟
به نظرتون moment میتونه کاری بکنه ؟

**Comment by AminRahimi** [comment]
مگه پارسر داره؟
پارسر یعنی تابعی که یک ورودی کاربر رو بگیره و تبدیلش کنه به تاریخ معتبر

**Comment by Sadeghbayan** [comment]
نمیدونم ، حقیقتش کار نکردم باهاش
خب اینجوری باشه من میتونم پارسر خودم رو تعریف کنم و تاریخ رو معتبر کنم ؟

**Comment by Sadeghbayan** [comment]
https://github.com/jalaali/moment-jalaali
این رو ببینید لطفا

**Comment by mohebifar** [comment]
Yes, `moment-jalali` is able to parse jalali formatted dates by giving `j` prefixed literals. like `moment('1939-02-02', 'jYYYY-jMM-jDD')`.

**Comment by Sadeghbayan** [comment]
https://github.com/Sadeghbayan/Angular-Directive-Persian-DatePickerJqueryUi#angular-directive-persian-datepickerjqueryui
